Sources/TensorFlow/Core/Runtime.swift

```diff
@@ -1201,6 +1201,17 @@ class _ThreadLocalState {
 
     var lazyTensorContext = LazyTensorContext()
 
+    static var useLazyTensor: Bool {
+        get {
+            _ThreadLocalState.local.lazyTensorEnabled ?? _RuntimeConfig.useLazyTensor
+        }
+        set(newValue) {
+            _ThreadLocalState.local.lazyTensorEnabled = newValue
+        }
+    }
+
+    private var lazyTensorEnabled: Bool? = nil
```

Comment:
Why is this optional?

Reply:
This may not always be set. If this is not set, we should use the global `_RuntimeConfig.useLazyTensor` property to determine if lazy tensor is enabled.  I added a comment to explain this.

Reply:
Hmm, isn't using the global `_RuntimeConfig.useLazyTensor` still not thread-safe? Would it make sense to make this flag the only one?